translations/harbour-fernschreiber-ru.ts

```diff
@@ -0,0 +1,655 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="ru_RU">
```

Comment:
Just `ru` should be enough, unless you're planning to submit a separate translation for e.g. `ru_UA`